faults.csv

```diff
@@ -1,63 +1,70 @@
-,,,,,,,,,,,,,,
\ No newline at end of file
+﻿Three Letter Code,Short Description,Long Description,Recommended Corrective Actions,Level,Button Type,Button Path,Button Macros,Rack,PV Prefix,PV Suffix,OK If Equal To,Faulted If Equal To,Severity,Generic Short Description for Decoder,
+OFF,Offline,Cavity not usable or not intended to be used for extended period,No further action required,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=10,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,HWMODE,,2,5,Offline,
+MNT,Maintenance,Maintenance: Cavity is known to be bad and is actively being worked on,No further action required,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=10,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,HWMODE,,1,4,Maintenance,
+MND,Maintenance Done,Maintenance done but cavity has not been run up for validation,"If program permits, set online and verify stability. If stable, leave online. If unstable, comment on/reopen the relevant CATER and set offline",CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=10,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,HWMODE,,3,4,Maintenance Done,
+RDY,Ready,Cavity usable but turned off for now,"If program permits, set online",CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=10,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,HWMODE,,4,4,Ready,
+BSO,BSOIC Tripped Chain A,BSOIC tripped,Communicate the fault to the EOIC and await resolution,ALL,EDM,$EDM/pps/pps_sysw.edl,,,BSOC:SYSW:2:,SumyA,1,,2,BSOIC Tripped,
+BSO,BSOIC Tripped Chain B,BSOIC tripped,Communicate the fault to the EOIC and await resolution,ALL,EDM,$EDM/pps/pps_sysw.edl,,,BSOC:SYSW:2:,SumyB,1,,2,BSOIC Tripped,
+PPS,Not Ready for Beam Chain A,PPS: West Global not in Ready for Beam,Communicate the fault to the EOIC and await resolution,ALL,EDM,$EDM/pps/pps_linacwest.edl,,,PPS:SYSW:1:,BeamReadyA,1,,2,Not Ready for Beam,
+PPS,Not Ready for Beam Chain B,PPS: West Global not in Ready for Beam,Communicate the fault to the EOIC and await resolution,ALL,EDM,$EDM/pps/pps_linacwest.edl,,,PPS:SYSW:1:,BeamReadyB,1,,2,Not Ready for Beam,
+HWI,Hardware Initialization needed,Hardware Initialization (a.k.a. rack checkout) needs to be run or other LLRF chassis problem. Don't trust current values.,Verify that the rack is not actively being worked on and run Rack Hardware Init and Test > Rack Checkout > Go,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=8,SELCHAR=3",A,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,HWINITSUM,,2,2,,
+HWI,Hardware Initialization needed,Hardware Initialization (a.k.a rack checkout) needs to be run or other LLRF chassis problem. Don't trust current values.,Verify that the rack is not actively being worked on and run Rack Hardware Init and Test > Rack Checkout > Go,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=8,SELCHAR=3",B,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,HWINITSUM,,2,2,,

```

Comment:
Ooh, now that we've got different fault codes to differentiate HWI and HWR, can we also update the last column (Generic Short Description for Decoder) to say "Hardware Initialization needed"?